.github/CHANGELOG.md

```diff
@@ -67,6 +67,9 @@
 
 ### Bug fixes
 
+* `lightning.qubit` and `lightning.kokkos` use `qml.ops.Conditional.base` instead of `qml.ops.Conditional.then_op`.
+  [(#752)](https://github.com/PennyLaneAI/pennylane-lightning/pull/752)
+
```

Comment:
Does this even need a changelog entry?

```suggestion
```

Reply:
You can never go wrong with a changelog.